rotkehlchen/tests/unit/test_evm_tx_decoding.py

```diff
@@ -327,3 +329,41 @@ def test_token_detection_after_decoding(
         assert save_tokens_mock.call_args_list[0].kwargs['address'] == ethereum_accounts[0]
         assert save_tokens_mock.call_args_list[0].kwargs['blockchain'] == SupportedBlockchain.ETHEREUM  # noqa: E501
         assert save_tokens_mock.call_args_list[0].kwargs['tokens'] == [Asset('eip155:1/erc20:0x98C23E9d8f34FEFb1B7BD6a91B7FF122F4e16F5c')]  # noqa: E501
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0xf4ae64c5c4fb632D0e0D77097b957941c399d26e']])
+def test_eip7702_transaction(ethereum_transaction_decoder, ethereum_accounts):
+    evmhash = deserialize_evm_tx_hash('0x42402dcf6658abaf2c47593a7ebe1264fb2f331de918239d1717a7a9d2996abf')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_transaction_decoder.evm_inquirer,
+        tx_hash=evmhash,
+    )
+    expected_events = [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=0,
+            timestamp=(timestamp := TimestampMS(1746615035000)),
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            amount=FVal(gas_amount := '0.0002516693954254'),
+            location_label=(user_address := ethereum_accounts[0]),
+            counterparty=CPT_GAS,
+            notes=f'Burn {gas_amount} ETH for gas',
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_ETH,
+            amount=ZERO,
+            location_label=user_address,
+            address=string_to_evm_address('0x775c8D470CC8d4530b8F233322480649f4FAb758'),
+            notes='Successfully executed eip-7702 transaction by delegate 0x775c8D470CC8d4530b8F233322480649f4FAb758',  # noqa: E501
```

Comment:
I find this confusing. Something better would be

```suggestion
            notes='Executed account delegation to 0x775c8D470CC8d4530b8F233322480649f4FAb758',  # noqa: E501
```